timesketch/api/v1/resources/event.py

```diff
@@ -53,21 +53,24 @@
 logger = logging.getLogger("timesketch.event_api")
 
 
-def _tag_event(row, tag_dict, tags_to_add, datastore, flush_interval):
-    """Tag each event from a dataframe with tags.
+def _tag_untag_event(row, tag_dict, tags_to_modify, datastore, flush_interval, tag):
+    """Tag or untag each event from a dataframe with tags.
 
     Args:
         row (np.Series): a single row of data with existing tags and
             information about the event in order to be able to add
             tags to it.
         tag_dict (dict): a dict that contains information to be returned
             by the API call to the user.
-        tags_to_add (list[str]): a list of strings of tags to add to each
-            event.
+        tags_to_modify (list[str]): a list of strings of tags to remove
+            or add on each event.
         datastore (opensearch.OpenSearchDataStore): the datastore object.
         flush_interval (int): the number of events to import before a bulk
             update is done with the datastore.
+        tag (bool): a boolean that decides if to tag or untag events.
```

Comment:
I do not have a really good name for this variable, other option was to make an "action" as a string and pass "add" or "remove" to it to flow control the method, but I was not happy with it either.

Reply:
I would recommend to go with the string / flag approach, since it makes understanding the code much more easy.

Reply:
done